core/templates/domain/question/pretest-question-backend-api.service.spec.ts

```diff
@@ -133,4 +133,19 @@ describe('Pretest question backend API service', function() {
       expect(failHandler).toHaveBeenCalled();
     })
   );
+
+  it('should call successHandler with [] when storyId is empty',
+    fakeAsync(() => {
+      var successHandler = jasmine.createSpy('success');
+      var failHandler = jasmine.createSpy('fail');
+
+      pretestQuestionBackendApiService.fetchPretestQuestions(
+        'expId', '').then(successHandler, failHandler);
```

Comment:
I think you can create a variable `storyId` above and then assign it to `''`. It'll be easier to understand the test itself.

Reply:
I created a variable called storyId and submitted the commit. Thank you so much for your assistance.